lib/libyaml-cpp.part

```diff
@@ -1,18 +1,22 @@
 Import('*')
-PartName("libyaml-cpp")
-PartVersion(GitVersionFromTag("0.6.3"))
+PartName("yaml-cpp")
+PartVersion(GitVersionFromTag("0.8.0"))
 
 env.AppendUnique(
     CPPPATH=["${CHECK_OUT_DIR}/include"],
     # add these to C++ compiler (CCFLAGS are for both c and C++, CFLAGS for C only)
-    CXXFLAGS=["-std=c++17","-fPIC"],
+    CXXFLAGS=["-std=c++17", "-g", "-fPIC"],
```

Comment:
Don't force debug information if it not defined in the config being used

Reply:
Thanks, good find.